pkg/controller/mizar/util_test.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mizar
+
+import (
+	"strconv"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	testAddress           = "test-address"
+	testHostIP            = "test-hostip"
+	testIP                = "test-ip"
+	testLabelNetworkValue = "test-network"
+	testName              = "test-name"
+	testNamespace         = "test-namespace"
+	testPhase             = "test-phase"
+	testProtocal          = "test-protocal"
+	testTenant            = "test-tenant"
+
+	testPort1 = 123
+	testPort2 = 4567
+)
+
+func TestConvertToServiceEndpointContract(t *testing.T) {
```

Comment:
it seems multi cases i ome test. Please use the pattern of golang subtest to restructure them.

Reply:
I checked and subtest is not fix for the 3 test in this file. The 3 test is not 3 different data for 1 type. It's 3 different types. The method to be tested are different.